component/loki/source/gcplog/internal/gcplogtarget/push_target.go

```diff
@@ -4,6 +4,12 @@ package gcplogtarget
 // configure and run the targets that can read log entries from cloud resource
 // logs like bucket logs, load balancer logs, and Kubernetes cluster logs
 // from GCP.
+// /!\ But
```

Comment:
We also need to take another shot at this; we're not running the server in the way that Promtail does, and we must not revert to it